### PR TITLE
ddtrace/tracer: add manual.keep and manual.drop tags

### PR DESCRIPTION
**Pull request by gbbr**
These are aliases for simplifying user-controlled sampling priorities.

### CODE REVIEW

File: ddtrace/ext/tags.go [hunk 1]
```diff
@@ -67,4 +67,12 @@ const (
 	// AnalyticsEvent specifies whether the span should be recorded as a Trace
 	// Search & Analytics event.
 	AnalyticsEvent = "analytics.event"
+
+	// ManualKeep is a tag which specifies that the trace to which this span
+	// belongs too should be kept when set to true.
```
Comment (cgilmour) [inline ddtrace/ext/tags.go]: `too` -> `to`

File: ddtrace/ext/tags.go [hunk 2]
```diff
@@ -67,4 +67,12 @@ const (
...
+	ManualKeep = "manual.keep"
+
+	// ManualDrop is a tag which specifies that the trace to which this span
+	// belongs too should be dropped when set to true.
```
Comment (cgilmour) [inline ddtrace/ext/tags.go]: `too` -> `to`

File: ddtrace/tracer/span.go [hunk 3]
```diff
@@ -158,6 +151,26 @@ func (s *span) setTagString(key, v string) {
...
+		}
+	case ext.ManualDrop:
+		if v {
+			s.setTagNumeric(ext.SamplingPriority, ext.PriorityUserReject)
+		}
+	case ext.ManualKeep:
+		if v {
+			s.setTagNumeric(ext.SamplingPriority, ext.PriorityUserKeep)
+		}
+	}
```
Comment (cgilmour) [inline ddtrace/tracer/span.go]: Does this need a `default` case?

File: ddtrace/tracer/span.go [hunk 4]
```diff
@@ -158,6 +151,26 @@ func (s *span) setTagString(key, v string) {
...
+// setTagBool sets a boolean tag on the span.
+func (s *span) setTagBool(key string, v bool) {
+	switch key {
+	case ext.AnalyticsEvent:
+		if v {
+			s.setTagNumeric(ext.EventSampleRate, 1.0)
+		} else {
+			s.setTagNumeric(ext.EventSampleRate, 0.0)
+		}
+	case ext.ManualDrop:
```
Comment (cgilmour) [inline ddtrace/tracer/span.go]: Is the tag supposed to be added to the span in addition to adjusting sampling priority's value?

Comment (gbbr) [reply ddtrace/tracer/span.go]: It's just a short-hand alias to simplify setting sampling priority, so no.

**Comment by gbbr** [comment]
Thank you too!

**Pull request merged**